src/MensaWidget/MWMenuBar.class.st

```diff
@@ -0,0 +1,234 @@
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:49'
+}
+MWMenuBar >> createDateDisplay [
+	
+	self dateDisplay: 
+		(TextMorph new 
+			contents: self date asString;
+			readOnly: true).
+	self addMorph: self dateDisplay
+	
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:52'
+}
+MWMenuBar >> createLayout [
+	
+	self createSpacer;
+		createNextButton;
+		createDateDisplay;
+		createPrevButton;
+		createSpacer
+	
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 16:06'
+}
+MWMenuBar >> createNextButton [
+
+	self addMorph: 
+		(self class getArrowButton
+			on: #mouseDown send: #increaseDate: to: self)
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 16:13'
+}
+MWMenuBar >> createPrevButton [
+	
+	self addMorph: 
+		(self class getArrowButton
+			rotationDegrees: 180;
+			on: #mouseDown send: #decreaseDate: to: self)
+	
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:50'
+}
+MWMenuBar >> createSpacer [
+	
+	self addMorph: 
+		(AlignmentMorph new 
+			hResizing: #spaceFill; 
+			vResizing: #spaceFill; 
+			color: Color transparent)
+	
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/7/2024 13:46'
+}
+MWMenuBar >> date [
+	
+	^ date
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/7/2024 13:46'
+}
+MWMenuBar >> date: anObject [
+	
+	date := anObject
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/7/2024 13:48'
+}
+MWMenuBar >> dateDisplay [
+
+	^ dateDisplay
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Louis Christ 6/7/2024 13:48'
+}
+MWMenuBar >> dateDisplay: anObject [
+
+	dateDisplay := anObject
+]
+
+{
+	#category : #dateManipulation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:50'
+}
+MWMenuBar >> decreaseDate [
+
+	self date: (self date addDays: -1);
+		updateDate
+]
+
+{
+	#category : #dateManipulation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 16:22'
+}
+MWMenuBar >> decreaseDate: aEvent [
+	
+	aEvent handler color: MWColorTheme backgroundColor.
+	World doOneCycle.
+	self decreaseDate.
+	aEvent handler color: MWColorTheme secondaryColor
+]
+
+{
+	#category : #dateManipulation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:50'
+}
+MWMenuBar >> increaseDate [
+
+	self date: (self date addDays: 1);
+		updateDate
+]
+
+{
+	#category : #dateManipulation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 16:22'
+}
+MWMenuBar >> increaseDate: aEvent [
+	
+	aEvent handler color: MWColorTheme backgroundColor.
+	World doOneCycle.
+	self increaseDate.
+	aEvent handler color: MWColorTheme secondaryColor
+]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:52'
+}
+MWMenuBar >> initialize [
+	
+	super initialize.
+	self date: Date today;
+		hResizing: #spaceFill;
+		vResizing: #shrinkWrap;
+		cellPositioning: #center;
+		color: Color paleGreen darker darker;
```

Comment:
I think it would be nice to have the color in a variable.

Reply:
in this case MWColorTheme primaryColor